### PR TITLE
OboeTester: Use AndroidX

### LINKED ISSUE #1740: Update OboeTester to use AndroidX
AndroidX is now recommended as the best practice

### PR DESCRIPTION
**Pull request by robertwu1**
Fixes #1740

### CODE REVIEW

File: apps/OboeTester/app/src/main/java/com/mobileer/oboetester/AnalyzerActivity.java [hunk 1]
```diff
@@ -78,27 +79,27 @@ protected String getCommonTestReport() {
         report.append("build.fingerprint = " + Build.FINGERPRINT + "\n");
         try {
             PackageInfo pinfo = getPackageManager().getPackageInfo(getPackageName(), 0);
-            report.append(String.format("test.version = %s\n", pinfo.versionName));
-            report.append(String.format("test.version.code = %d\n", pinfo.versionCode));
+            report.append(String.format(Locale.getDefault(), "test.version = %s\n", pinfo.versionName));
```
Comment (philburk) [inline apps/OboeTester/app/src/main/java/com/mobileer/oboetester/AnalyzerActivity.java:82]: You don't need to pass Locale.getDefault(). That is the default.
https://docs.oracle.com/javase/7/docs/api/java/lang/String.html#format(java.lang.String,%20java.lang.Object...)

**Review by philburk — CHANGES_REQUESTED** [review]

File: apps/OboeTester/app/src/main/java/com/mobileer/oboetester/AnalyzerActivity.java (hunk 1 above)
Comment (robertwu1) [reply apps/OboeTester/app/src/main/java/com/mobileer/oboetester/AnalyzerActivity.java:82]: You get the warning `
Implicitly using the default locale is a common source of bugs: Use String.format(Locale, ...) instead` if you don't do this

Comment (robertwu1) [reply apps/OboeTester/app/src/main/java/com/mobileer/oboetester/AnalyzerActivity.java]: Updated to Locale.US

**Review by philburk — APPROVED** [review]
This makes the code look messy without adding any functionality.
But it gets rid of an annoying Android Studio warning.
And it helps Google3.
OK.

**Pull request merged**